sass/understrap/understrap.scss

```diff
@@ -42,18 +42,18 @@
 .alignleft {
 	display: inline;
 	float: left;
-	margin-right: 1.5em;
+	margin-right: $spacer;
```

Comment:
By default $spacer is 1rem which probably is not equal to 1.5em. This changes frontend styles which are not necessarily related to Gutenberg.